src/test/script_tests.cpp

```diff
@@ -40,7 +40,7 @@ using namespace json_spirit;
 using namespace boost::algorithm;
 
 // Uncomment if you want to output updated JSON tests.
-// #define UPDATE_JSON_TESTS
+#define UPDATE_JSON_TESTS
```

Comment:
Woops, that uncommenting of the JSON test regeneration is unintentional.
On Jan 30, 2015 10:49 AM, "Luke-Jr" <notifications@github.com> wrote:

> In src/test/script_tests.cpp
> <https://github.com/bitcoin/bitcoin/pull/5714#discussion_r23846306>:
>
> > @@ -40,7 +40,7 @@ using namespace json_spirit;
> >  using namespace boost::algorithm;
> >
> >  // Uncomment if you want to output updated JSON tests.
> > -// #define UPDATE_JSON_TESTS
> > +#define UPDATE_JSON_TESTS
>
> FWIW, I also noticed this but since I didn't see the generated files in
> the git repository, I concluded it sense.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/5714/files#r23846306>.
>